src/app/core/market/order-status-notifier/order-status-notifier.service.ts

```diff
@@ -35,56 +36,64 @@ export class OrderStatusNotifierService implements OnDestroy {
     if (orders.orders.length === 0) {
       return;
     }
-
-    if (!this.oldOrders || this.oldOrders.length === 0) {
-      this.oldOrders = orders.bid.orders;
-    } else {
-      this.checkOrders(orders.bid.orders);
-      this.oldOrders = orders.bid.orders;
+    if (this.oldOrders && this.oldOrders.length) {
+      this.checkOrders(orders.bid.orders.reverse());
     }
+    this.oldOrders = orders.bid.orders.reverse();
```

Comment:
Is this intended reverse behavior? 
https://www.w3schools.com/jsref/jsref_reverse.asp

reverse() does not return a new array with the reversed order.
It reverses the array you are calling it on.

Reply:
hmm... yeah need to reverse first and then assign :)